Makefile.PL

```diff
@@ -8,9 +8,14 @@ use ExtUtils::MakeMaker;
 use File::Spec::Functions qw( catfile rel2abs );
 use Getopt::Long qw( GetOptionsFromArray );
 
-push @INC, '.';
-unless( require( catfile qw(inc IO Interactive Tiny.pm) ) ) {
-    die 'Your distribution is incomplete: Failed to load bundled IO::Interactive::Tiny';
+BEGIN {
+  # Must use a bundled version
+  # "./" prefix required to subvert @INC traversal
+  # "catfile" not useful here as it eats "." and may produces wrong \
+  #  under windows ( require always takes / )
+  local $@;
+  eval { require "./inc/IO/Interactive/Tiny.pm" ; 1 } or
+    die "Your distribution is incomplete: Failed to load bundled IO::Interactive::Tiny\n$@";
```

Comment:
That `;1` inside the `eval{}` is superfluous since `require` never returns a false value.